### PR TITLE
Readme documentation

### PR DESCRIPTION
**Pull request by spurohitsynopsys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,88 +19,141 @@ Synopsys Security Scan supports Azure integration with the following Synopsys se
...
+| `BRIDGE_POLARIS_ASSESSMENT_TYPES` | Polaris assessment types. Example: SCA,SAST                       | Mandatory          |
 
 # Synopsys Security Scan - Coverity Cloud Deployment with Thin Client
 
 At this time, Synopsys Security Scan only supports the Coverity thin client/cloud deployment model, which removes the need for a large footprint installation in your agent.
 
 Before running Coverity using the Synopsys Security Scan, ensure the appropriate `project` and `stream` are set in your Coverity Connect server environment.
 
-Configure sensitive data like usernames, passwords and URLs using pipeline variables.
+Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Coverity into azure pipeline.
```
Comment (sig-tithi) [inline README.md:91]: azure should be Azure. check for other places as well.

File: README.md [hunk 2]
```diff
@@ -109,55 +162,72 @@ trigger:
...
-
-# Azure Agent Setup
-
-- Agents can be installed and used on GNU/Linux, macOS, Windows and Docker. See this documentation for details:
-https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=browser
-- You can use Microsoft-hosted agents as well to scan your code using Azure Pipelines.
+|`SYNOPSYS_BRIDGE_PATH`| Provide a path, where you want to configure or already configured Synopsys Bridge.<br/> [Note - If you don't provide any path, then by default configuration path will be considered as - $HOME/synopsys-bridge].<br/> If the configured Synopsys Bridge is not the latest one, latest Synopsys Bridge version will be downloaded          | Optional     |
+| `BRIDGE_DOWNLOAD_URL`      | Provide URL to bridge zip file.<br/> If provided, Synopsys Bridge will be automatically downloaded and configured in the provided bridge- or default- path.<br/> [Note - As per current behavior, when this value is provided, the bridge_path or default path will be cleaned first then download and configured all the time]               | Optional     |
```
Comment (sig-tithi) [inline README.md:226]: [Note - As per current behavior, when this value is provided, the bridge_path or default path will be cleaned first then download and configured all the time]            - just confirm it's current behaviour

Comment (lsynopsys) [reply README.md:226]: can we say cleared instead of cleaned

Comment (spurohitsynopsys) [reply README.md:226]: i think we can remove this part, we have removed this in Gitlab as well

File: README.md (hunk 1 above)
Comment (spurohitsynopsys) [reply README.md]: updated

**Review by lsynopsys — APPROVED** [review]

**Comment by sig-tithi** [comment]
approved. don't see the approval option.

**Pull request merged**